acm/odh-edge/acm-observability/files/uwm_metrics.yaml

```diff
@@ -8,3 +9,19 @@
       - rest_server_request_duration_seconds_count
       - rest_server_requests_in_progress
       - rest_server_requests_total
+
+      # OpenVINO metrics
```

Comment:
Do you think it'd be useful to add comments with the metric type shown [in this gist](https://gist.github.com/grdryn/3fccf4805273b9ec91c135215e3dd0b9)

Reply:
I don't _think_ that would be useful, since the source of truth for that information is the metrics endpoint (and gets pulled into the Prometheus server too, I think), and it's discoverable from there. If we add it statically here, then if there are changes, this would be out-of-date/incorrect.

I'm happy to add it if that's the consensus though.

Reply:
it's not a showstopper for me, and I understand your concern about the source of truth but you can also say the same thing about the metrics names.

I feel it'd help when creating the dashboards or just trying to understand the metrics without having a running service. Documenting metrics names and types is a pretty common practice. But as I said, it's not a show stopper and since this is merged you can put your efforts else where now.

Reply:
If it's a common practice, then we should copy the same pattern as the other places where it's common. If you can link one or more places as an example of which bits of information is expected and in what format, we can look to work towards that! :+1: